query/security/common/src/main/java/org/codice/ddf/admin/security/common/fields/wcpm/services/PolicyManagerServiceProperties.java

```diff
@@ -75,7 +75,7 @@ public ContextPolicies policyManagerSettingsToBins(PolicyManager policyManager)
             }
         }
 
-        ContextPolicies policiesField = new ContextPolicies();
+        ListField<ContextPolicyBin> policiesField = new ListFieldImpl<>(ContextPolicyBin.class);
```

Comment:
Need fieldName here? Default is null right now, so `fieldName()` will return null.